hub/src/main/java/org/motechproject/hub/service/SubscriptionService.java

```diff
@@ -35,6 +35,8 @@
      *            be used to compute an HMAC digest for authorized content
      *            distribution. Currently this is not being consumed by the API
      * @throws HubException
+     *            - if tries unsubscribing from not subscribed or non existing topic or
+     *            topic was not found from any other reason
```

Comment:
when unsubscribing from a topic that was not subscribed to, when the topic does not exist or when the topic was not found